src/NHibernate/Cfg/SettingsFactory.cs

```diff
@@ -268,11 +268,6 @@ public Settings BuildSettings(IDictionary<string, string> properties)
 			log.Info("Default flush mode: " + defaultFlushMode);
 			settings.DefaultFlushMode = defaultFlushMode;
 
-			EntityMode defaultEntityMode =
-				EntityModeHelper.Parse(PropertiesHelper.GetString(Environment.DefaultEntityMode, properties, "poco"));
-			log.Info("Default entity-mode: " + defaultEntityMode);
```

Comment:
Here, it's better to warn that default entity mode is deprecated rather then silently remove it.